dim_checks/moz-fx-data-shared-prod/telemetry/unified_metrics/dim_checks.yaml

```diff
@@ -0,0 +1,95 @@
+dim_config:
+  owner:
+    email: lvargas@mozilla.com
+    slack: lvargas
+  alerts_enabled:
+    enabled: true
+    notify:
+      channels:
+        - dim_test_run
+  tier: tier_1
+  dim_tests:
+    - type: table_row_count
+      params:
+        condition: ">= 1000"
+    - type: not_null
+      params:
+        columns:
```

Comment:
`unified_metrics_v1` has 37 columns excluding `sample_id`. Actually all of them are NULLABLE but I'd recommend a second check with Leif(?) about the most relevant columns to monitor NULL values. I find them all informative, but some may generate an immediate alarm e.g. client_id, while others are expected and perhaps not that urgent e.g. city.

Reply:
@irrationalagent Do you have any thoughts about this?

Reply:
client_id should always be present, and i think of a certain length also but we may need to check if there are platform-based differences there. 

sample_id is derived from client_id, but it is not present in `telemetry.core_clients_last_seen` so we need to keep that nullable i think. 

`days_seen_bits` is computed by the pipeline and so **should** always be present, but I cannot confirm this 100% (will need to write a query to see for myself)

for the rest, I can see plausible, if infrequent, cases where they might be null.

Reply:
I'll keep the rest as is for now and we can always adjust the config over time.